tests/standalone/test-imagecdn.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * This file contains standalone tests for the ImageCDN functionality
+ *
+ * @package ImageCDN
+ */
+
+use ImageEngine\ImageCDN;
+
+/**
+ * The ImageCDNTest class tests the ImageCDN class.
+ */
+class ImageCDNTest extends PHPUnit_Framework_TestCase {
+
+    public function testAddHeaders() {
+        ImageCDN::$tests_running = true;
+        ImageCDN::$test_headers_written = [];
+        ImageCDN::$test_options = [
+            'url' => 'https://foo.com',
+        ];
+
+        ImageCDN::add_headers();
+
+        $expected = [
+            'Accept-CH: viewport-width, width, dpr, ect',
+            'Link: <https://foo.com>; rel=preconnect',
+            'Feature-Policy: ch-viewport-width https://foo.com; ch-width https://foo.com; ch-dpr https://foo.com; ch-ect https://foo.com',
+            'Permissions-Policy: ch-viewport-width=("https://foo.com"), ch-width=("https://foo.com"), ch-dpr=("https://foo.com"), ch-ect=("https://foo.com")',
```

Comment:
@jonarnes this test should help us prevent regressions as we make changes to these headers in the future.

You can run these standalone tests like this:
```
# Make sure the testing dependencies are installed
composer install

# Run the tests
vendor/bin/phpunit -vvv -c phpunit-standalone.xml.dist 

Output:
PHPUnit 5.7.27 by Sebastian Bergmann and contributors.

Runtime:       PHP 7.4.3 with Xdebug 2.9.2
Configuration: /home/steve/Workspace/image-cdn-wordpress/phpunit-standalone.xml.dist

...........                                                       11 / 11 (100%)

Time: 27 ms, Memory: 8.00MB

OK (11 tests, 32 assertions)
```